build/docker-entrypoint.sh

```diff
@@ -1,19 +1,27 @@
 #!/usr/bin/env bash
 
-if [ $1 == "dst-server" ]; then
-	set -e
+# Exit immediately on non-zero return codes.
+set -e
```

Comment:
I think that this doesn't have any bad or good. With exec as soon as the process ends the container ends too. The exit code of the process doesn't change that. So we can keep it outside.